### PR TITLE
Added get album for Watson

### PR DESCRIPTION
**Pull request by pnavk99**

### CODE REVIEW

File: envs/dev-windows.bat [hunk 1]
```diff
@@ -3,3 +3,4 @@ set FLASK_ENV=development
 set SECRET_KEY=dev
 set SPOTIPY_CLIENT_ID="5d141ce6f09c4fbfa12d16ce9e5d40c1"
 set SPOTIPY_CLIENT_SECRET="52fed0d6564a4e6f8e596b78bd1abf62"
+set DISCOVERY_API_KEY="L8vZ5k9U8L8r8r9YstxDbOml0imvZBYmj7bB_tiVQ2GS"
```
Comment (Alex7Li) [inline envs/dev-windows.bat:6]: Remove the doublequotes from this line, I was getting an error about quotes. I guess windows interprets the files differently and maybe spotipy doesn't care

**Review by Alex7Li — CHANGES_REQUESTED** [review]
Looks good, when I loaded it up on windows I got an error though

File: envs/dev-windows.bat (hunk 1 above)
Comment (pnavk99) [reply envs/dev-windows.bat]: Done

**Review by Alex7Li — APPROVED** [review]
LGTM, not sure if you want sam to review or not so I won't merge

**Pull request merged**